updateUserFeatureFlag/main.go

```diff
@@ -0,0 +1,133 @@
+package main
+
+import (
+	"encoding/json"
+	"log"
+	"net/http"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/Real-Dev-Squad/feature-flag-backend/database"
+	"github.com/Real-Dev-Squad/feature-flag-backend/models"
+	"github.com/Real-Dev-Squad/feature-flag-backend/utils"
+	"github.com/aws/aws-lambda-go/events"
+	"github.com/aws/aws-lambda-go/lambda"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/service/dynamodb"
+	"github.com/aws/aws-sdk-go/service/dynamodb/dynamodbattribute"
+	"github.com/go-playground/validator/v10"
+)
+
+var validate *validator.Validate
+
+func init() {
+	validate = validator.New()
+}
+
+func processUpdateByIds(userId string, flagId string, requestBody models.UpdateUserMapping) (*models.FeatureFlagUserMapping, error) {
+
+	db := database.CreateDynamoDB()
+
+	input := &dynamodb.UpdateItemInput{
+		TableName: aws.String(database.GetTableName(utils.FFUM_TABLE_NAME)),
+		Key: map[string]*dynamodb.AttributeValue{
+			utils.UserId: {
+				S: aws.String(userId),
+			},
+			utils.FlagId: {
+				S: aws.String(flagId),
+			},
+		},
+		UpdateExpression: aws.String("set #status = :status, #updatedBy = :updatedBy, #updatedAt = :updatedAt"),
+		ExpressionAttributeNames: map[string]*string{
+			"#status":    aws.String("status"),
+			"#updatedBy": aws.String("updatedBy"),
+			"#updatedAt": aws.String("updatedAt"),
```

Comment:
why do we have these `#` symbols?

Reply:
All these are [expression attribute names](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/Expressions.ExpressionAttributeNames.html) and as per syntax, they should start with a `#` symbol. Since `status` is a reserved keyword we are using expression attribute names and for consistency and safety we use it for all other attributes as well